packages/common/src/reducers/messages.js

```diff
@@ -231,6 +231,7 @@ const deriveMessageType = (message) => {
     case 1:
       return "member-joined";
     case 2:
+    case 3:
```

Comment:
Forgot to add this to handle other app message types. Might change later to it's own type, if we decide to handle apps vs. pure webhooks differently